ui/ceval/src/ctrl.ts

```diff
@@ -231,6 +231,7 @@ export default function (opts: CevalOpts): CevalCtrl {
             downloadProgress(mb);
             opts.redraw();
           }),
+          // TODO: Update version to the new asset on CDN
           version: '2732f2',
```

Comment:
For staging/production, this hard-coded version has to be aligned to the new asset on CDN.

Reply:
You can pick any random 6 letters.

Reply:
I thought that the version string is generated somewhere automatically. Is that not the case?

Reply:
Okay, I updated it with the random number I got from `openssl rand -hex 3`.

Reply:
In nginx it's really just:

    location /assets/ {
      # ...
      rewrite "^/assets/_\w{6}/(.+)$" /assets/$1;
      alias /home/lichess/public/;
    }

Reply:
Oh, I understand now. So the point is that just to fix the url irregardless of assets deployment. Thanks for the clarification.